src/CSET/operators/plot.py

```diff
@@ -656,14 +781,119 @@ def plot_vertical_line_series(
     if filename is None:
         filename = slugify(recipe_title)
 
-    # Make vertical line plot
-    plotting_func = _plot_and_save_vertical_line_series
-
     # set the lower and upper limit for the x-axis to ensure all plots
     # have same range. This needs to read the whole cube over the range of
     # the sequence and if applicable postage stamp coordinate.
     # This only works if the plotting is done in the collate section of a
     # recipe and not in the parallel section of a recipe.
+    vmin = np.floor(cube.data.min())
+    vmax = np.ceil(cube.data.max())
+
+    # Create a plot for each value of the sequence coordinate.
+    plot_index = []
+    for cube_slice in cube.slices_over(sequence_coordinate):
+        # Use sequence value so multiple sequences can merge.
+        seq_coord = cube_slice.coord(sequence_coordinate)
+        sequence_value = seq_coord.points[0]
+        plot_filename = f"{filename.rsplit('.', 1)[0]}_{sequence_value}.png"
+        # Format the coordinate value in a unit appropriate way.
+        title = f"{recipe_title} | {seq_coord.units.title(sequence_value)}"
+        # Do the actual plotting.
+        _plot_and_save_vertical_line_series(
+            cube_slice,
+            coord,
+            plot_filename,
+            series_coordinate,
+            title=title,
+            vmin=vmin,
+            vmax=vmax,
+        )
+        plot_index.append(plot_filename)
+
+    # Add list of plots to plot metadata.
+    complete_plot_index = _append_to_plot_index(plot_index)
+
+    # Make a page to display the plots.
+    _make_plot_html_page(complete_plot_index)
+
+    return cube
+
+
+def plot_histogram_series(
+    cube: iris.cube.Cube,
+    filename: str = None,
+    sequence_coordinate: str = "time",
```

Comment:
as with previous comments around configurable line vs. bar

Reply:
thanks, added in 8ccc6d9..fd08bc0